src/functions/searches/attach-to-one-search/attach-to-one-search.ts

```diff
@@ -424,6 +418,7 @@ export const makeAttachToOneSearch = (context: APIContext) => {
 
 		filter$.subscribe(filter => {
 			requestEntries(filter);
+			setTimeout(() => requestEntries(filter), 2000); // TODO: Change this
```

Comment:
Removing this line causes tests to fail for me, as well as a race around the search subscription's `errors$`

Reply:
How about we remove the `requestEntries(filter);`, only keeping the `setTimeout(() => requestEntries(filter), 2000); // TODO`.

And also, we change from `2000` to `1000`. That way, it'll be the `debounceOptions.initialDueTime`

Reply:
We'll tackle that later. See #244